web/models/health_summary.go

```diff
@@ -0,0 +1,18 @@
+package models
+
+const (
+	HealthSummaryHealthPassing  = "passing"
+	HealthSummaryHealthWarning  = "warning"
+	HealthSummaryHealthCritical = "critical"
+	HealthSummaryHealthUnknown  = "unknown"
+)
+
+type HealthSummary []SAPSystemHealthSummary
+
+type SAPSystemHealthSummary struct {
+	ID             string `json:"id"`
+	SID            string `json:"sid"`
+	ClustersHealth string `json:"clusters_health"`
+	DatabaseHealth string `json:"databases_health"`
```

Comment:
Maybe `DatabasesHealth` better (last `s` missing). Or remove the `s` from `databases_health` as we will only have 1 database per system

Reply:
i blame copilot, fixed, thanks!